hack/attach-gdb.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# this script tries to attach to a gdbserver
+
+# wait for gdbserver to start
+while ! pidof gdbserver >& /dev/null; do sleep 1; done
+nsenter <&0 -t $(pgrep gdbserver) -n gdb $@
```

Comment:
I'd say using `pidof` instead of `pgrep` could be slightly more correct there. Also, please use `"$@"` instead of just `$@` (without quotes). In this particular case, that hardly matters much, but in other cases, `$@` will not handle `"argument values with spaces"` correctly